extensions/inline/sub.rng

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<grammar ns="http://libero.pub" xmlns="http://relaxng.org/ns/structure/1.0"
+    datatypeLibrary="http://www.w3.org/2001/XMLSchema-datatypes"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+
+    <sch:ns prefix="libero" uri="http://libero.pub"/>
+
+    <define name="libero.inline.full.highlighting" combine="choice">
```

Comment:
These aren't highlightling, suggest we either follow JATS and use `subsup`, or maybe `positioning` is better? (Maybe sounds too broad though.)